tests/unit/askPresentIdeasFromTeam.test.ts

```diff
@@ -98,6 +105,41 @@ describe("askPresentIdeasFromTeam", () => {
     });
   });
 
+  it("Should publish askPresentAndSquadJoinFromTeam event with a schedule", async () => {
+    const event = {
+      birthdayPerson: constants.userId,
+      team: constants.teamId,
+      eventId: constants.eventId,
+    } satisfies Events["askPresentIdeasFromTeam"];
```

Comment:
Shouldn't this be askPresentAndSquadJoinFromTeam?

Reply:
No, im testing here that the askPresentIdeasFromTeam is creating a schedule or not.